optimum/exporters/onnx/_tensor_cache.py

```diff
@@ -0,0 +1,95 @@
+from typing import Any, Dict, Optional, Tuple
+
+import torch
+from transformers import logging
+
+
+logger = logging.get_logger(__name__)
+
+
+# The same as transformers.cache_utils.Cache but iherits from torch.Tensor instead of torch.nn.Module
+class Cache(torch.Tensor):
```

Comment:
this is to make the cache classes traceable by torch script, it also makes more sense so I'm gonna propose this in the transformers lib as `nn.Module` was only proposed for compatibility with torch.export

Reply:
https://huggingface.slack.com/archives/C014N4749J9/p1737370244628979